internal/binder/function/funcs_obj.go

```diff
@@ -240,4 +240,56 @@ func registerObjectFunc() {
 		},
 		check: returnNilIfHasAnyNil,
 	}
+	builtins["object_pick"] = builtinFunc{
+		fType: ast.FuncTypeScalar,
+		exec: func(ctx api.FunctionContext, args []interface{}) (interface{}, bool) {
+			contains := func(array []string, target string) bool {
```

Comment:
Suggest to move this out as a common function instead of define it each time.

Reply:
Yes, moved it out in the same file. Since there is no more usage in other places.